Windows build Some keys not working e.g. backspace

### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

I have a very strange behavior. I compiled zed using mingw1310_64 with
`cargo run --release`

The window opens and I see the Welcome to Zed! intro screen. 
I disabled vim mode, opened a new file and begin to write.
If I hit backspace or delete, nothing happens. The character stay.
Also Ctrl+s doesn't do anything.

Next test was to enable vim mode. Then I am in NORMAL mode, but can't enter INSERT mode via i or a
I'm not even able to enter anything.
Back to disabled vim mode yields the same behavior as before enabling vim mode.

On closing zed, it'll ask for a filename to save.



### Environment

Opening the command palette using Ctrl+Shift+P doesn't work either. The backspace also doesn't work in command palette.
![image](https://github.com/user-attachments/assets/dfdca389-39d9-41f2-a435-405721fb2de1)


### If applicable, add mockups / screenshots to help explain present your vision of the feature

_No response_

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>
2024-08-14T13:35:03.4090841+02:00 [INFO] ========== starting zed ==========
2024-08-14T13:35:03.5717257+02:00 [INFO] Use Segoe UI as UI font.
2024-08-14T13:35:03.5735603+02:00 [INFO] Opening main db
2024-08-14T13:35:03.584468+02:00 [INFO] Using git binary path: None
2024-08-14T13:35:03.5856349+02:00 [ERROR] SHELL environment variable is not assigned so we can't source login environment variables

Caused by:
    environment variable not found
2024-08-14T13:35:03.5886198+02:00 [ERROR] invalid binding value for keystroke alt-enter, context None

Caused by:
    no action type registered for picker::ConfirmInput
2024-08-14T13:35:03.5890432+02:00 [ERROR] invalid binding value for keystroke ctrl-+, context None

Caused by:
    no action type registered for zed::IncreaseBufferFontSize
2024-08-14T13:35:03.5895301+02:00 [ERROR] invalid binding value for keystroke ctrl-,, context None

Caused by:
    no action type registered for zed::OpenSettings
2024-08-14T13:35:03.5897006+02:00 [ERROR] invalid binding value for keystroke ctrl--, context None

Caused by:
    no action type registered for zed::DecreaseBufferFontSize
2024-08-14T13:35:03.5898592+02:00 [ERROR] invalid binding value for keystroke ctrl-0, context None

Caused by:
    no action type registered for zed::ResetBufferFontSize
2024-08-14T13:35:03.5900221+02:00 [ERROR] invalid binding value for keystroke ctrl-=, context None

Caused by:
    no action type registered for zed::IncreaseBufferFontSize
2024-08-14T13:35:03.5901755+02:00 [ERROR] invalid binding value for keystroke ctrl-alt-enter, context None

Caused by:
    no action type registered for picker::ConfirmInput
2024-08-14T13:35:03.5903264+02:00 [ERROR] invalid binding value for keystroke ctrl-c, context None

Caused by:
    no action type registered for menu::Cancel
2024-08-14T13:35:03.5904787+02:00 [ERROR] invalid binding value for keystroke ctrl-enter, context None

Caused by:
    no action type registered for menu::SecondaryConfirm
2024-08-14T13:35:03.5906317+02:00 [ERROR] invalid binding value for keystroke ctrl-escape, context None

Caused by:
    no action type registered for menu::Cancel
2024-08-14T13:35:03.5907799+02:00 [ERROR] invalid binding value for keystroke ctrl-n, context None

Caused by:
    no action type registered for menu::SelectNext
2024-08-14T13:35:03.5909269+02:00 [ERROR] invalid binding value for keystroke ctrl-o, context None

Caused by:
    no action type registered for workspace::Open
2024-08-14T13:35:03.591076+02:00 [ERROR] invalid binding value for keystroke ctrl-p, context None

Caused by:
    no action type registered for menu::SelectPrev
2024-08-14T13:35:03.5912261+02:00 [ERROR] invalid binding value for keystroke ctrl-q, context None

Caused by:
    no action type registered for zed::Quit
2024-08-14T13:35:03.5913702+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-w, context None

Caused by:
    no action type registered for workspace::CloseWindow
2024-08-14T13:35:03.5915157+02:00 [ERROR] invalid binding value for keystroke end, context None

Caused by:
    no action type registered for menu::SelectLast
2024-08-14T13:35:03.5916581+02:00 [ERROR] invalid binding value for keystroke enter, context None

Caused by:
    no action type registered for menu::Confirm
2024-08-14T13:35:03.5918045+02:00 [ERROR] invalid binding value for keystroke escape, context None

Caused by:
    no action type registered for menu::Cancel
2024-08-14T13:35:03.5919486+02:00 [ERROR] invalid binding value for keystroke f11, context None

Caused by:
    no action type registered for zed::ToggleFullScreen
2024-08-14T13:35:03.5920889+02:00 [ERROR] invalid binding value for keystroke home, context None

Caused by:
    no action type registered for menu::SelectFirst
2024-08-14T13:35:03.5922298+02:00 [ERROR] invalid binding value for keystroke pagedown, context None

Caused by:
    no action type registered for menu::SelectLast
2024-08-14T13:35:03.5923737+02:00 [ERROR] invalid binding value for keystroke pageup, context None

Caused by:
    no action type registered for menu::SelectFirst
2024-08-14T13:35:03.5925187+02:00 [ERROR] invalid binding value for keystroke shift-escape, context None

Caused by:
    no action type registered for workspace::ToggleZoom
2024-08-14T13:35:03.592696+02:00 [ERROR] invalid binding value for keystroke shift-pagedown, context None

Caused by:
    no action type registered for menu::SelectFirst
2024-08-14T13:35:03.5928395+02:00 [ERROR] invalid binding value for keystroke shift-pageup, context None

Caused by:
    no action type registered for menu::SelectFirst
2024-08-14T13:35:03.5929862+02:00 [ERROR] invalid binding value for keystroke shift-tab, context None

Caused by:
    no action type registered for menu::SelectPrev
2024-08-14T13:35:03.5931286+02:00 [ERROR] invalid binding value for keystroke tab, context None

Caused by:
    no action type registered for menu::SelectNext
2024-08-14T13:35:03.5932814+02:00 [ERROR] invalid binding value for keystroke down, context Some("Picker || menu")

Caused by:
    no action type registered for menu::SelectNext
2024-08-14T13:35:03.5934272+02:00 [ERROR] invalid binding value for keystroke up, context Some("Picker || menu")

Caused by:
    no action type registered for menu::SelectPrev
2024-08-14T13:35:03.593569+02:00 [ERROR] invalid binding value for keystroke left, context Some("Prompt")

Caused by:
    no action type registered for menu::SelectPrev
2024-08-14T13:35:03.5937094+02:00 [ERROR] invalid binding value for keystroke right, context Some("Prompt")

Caused by:
    no action type registered for menu::SelectNext
2024-08-14T13:35:03.5938519+02:00 [ERROR] invalid binding value for keystroke alt-g b, context Some("Editor")

Caused by:
    no action type registered for editor::ToggleGitBlame
2024-08-14T13:35:03.5940003+02:00 [ERROR] invalid binding value for keystroke alt-pagedown, context Some("Editor")

Caused by:
    no action type registered for editor::PageDown
2024-08-14T13:35:03.5941414+02:00 [ERROR] invalid binding value for keystroke alt-pageup, context Some("Editor")

Caused by:
    no action type registered for editor::PageUp
2024-08-14T13:35:03.5942801+02:00 [ERROR] invalid binding value for keystroke backspace, context Some("Editor")

Caused by:
    no action type registered for editor::Backspace
2024-08-14T13:35:03.5944194+02:00 [ERROR] invalid binding value for keystroke ctrl-", context Some("Editor")

Caused by:
    no action type registered for editor::ExpandAllHunkDiffs
2024-08-14T13:35:03.5945562+02:00 [ERROR] invalid binding value for keystroke ctrl-', context Some("Editor")

Caused by:
    no action type registered for editor::ToggleHunkDiff
2024-08-14T13:35:03.5946942+02:00 [ERROR] invalid binding value for keystroke ctrl-;, context Some("Editor")

Caused by:
    no action type registered for editor::ToggleLineNumbers
2024-08-14T13:35:03.5948342+02:00 [ERROR] invalid binding value for keystroke ctrl-a, context Some("Editor")

Caused by:
    no action type registered for editor::SelectAll
2024-08-14T13:35:03.5949731+02:00 [ERROR] invalid binding value for keystroke ctrl-alt-space, context Some("Editor")

Caused by:
    no action type registered for editor::ShowCharacterPalette
2024-08-14T13:35:03.5951132+02:00 [ERROR] invalid binding value for keystroke ctrl-backspace, context Some("Editor")

Caused by:
    no action type registered for editor::DeleteToPreviousWordStart
2024-08-14T13:35:03.5952506+02:00 [ERROR] invalid binding value for keystroke ctrl-c, context Some("Editor")

Caused by:
    no action type registered for editor::Copy
2024-08-14T13:35:03.5953865+02:00 [ERROR] invalid binding value for keystroke ctrl-delete, context Some("Editor")

Caused by:
    no action type registered for editor::DeleteToNextWordEnd
2024-08-14T13:35:03.5955252+02:00 [ERROR] invalid binding value for keystroke ctrl-down, context Some("Editor")

Caused by:
    no action type registered for editor::LineDown
2024-08-14T13:35:03.5956641+02:00 [ERROR] invalid binding value for keystroke ctrl-end, context Some("Editor")

Caused by:
    no action type registered for editor::MoveToEnd
2024-08-14T13:35:03.5957977+02:00 [ERROR] invalid binding value for keystroke ctrl-home, context Some("Editor")

Caused by:
    no action type registered for editor::MoveToBeginning
2024-08-14T13:35:03.5959654+02:00 [ERROR] invalid binding value for keystroke ctrl-i, context Some("Editor")

Caused by:
    no action type registered for editor::ShowSignatureHelp
2024-08-14T13:35:03.5961037+02:00 [ERROR] invalid binding value for keystroke ctrl-insert, context Some("Editor")

Caused by:
    no action type registered for editor::Copy
2024-08-14T13:35:03.5962413+02:00 [ERROR] invalid binding value for keystroke ctrl-k, context Some("Editor")

Caused by:
    no action type registered for editor::CutToEndOfLine
2024-08-14T13:35:03.5963766+02:00 [ERROR] invalid binding value for keystroke ctrl-k ctrl-r, context Some("Editor")

Caused by:
    no action type registered for editor::RevertSelectedHunks
2024-08-14T13:35:03.596512+02:00 [ERROR] invalid binding value for keystroke ctrl-l, context Some("Editor")

Caused by:
    no action type registered for editor::SelectLine
2024-08-14T13:35:03.596652+02:00 [ERROR] invalid binding value for keystroke ctrl-left, context Some("Editor")

Caused by:
    no action type registered for editor::MoveToPreviousWordStart
2024-08-14T13:35:03.5967877+02:00 [ERROR] invalid binding value for keystroke ctrl-right, context Some("Editor")

Caused by:
    no action type registered for editor::MoveToNextWordEnd
2024-08-14T13:35:03.5969246+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-end, context Some("Editor")

Caused by:
    no action type registered for editor::SelectToEnd
2024-08-14T13:35:03.5970612+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-home, context Some("Editor")

Caused by:
    no action type registered for editor::SelectToBeginning
2024-08-14T13:35:03.5972055+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-i, context Some("Editor")

Caused by:
    no action type registered for editor::Format
2024-08-14T13:35:03.5973434+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-left, context Some("Editor")

Caused by:
    no action type registered for editor::SelectToPreviousWordStart
2024-08-14T13:35:03.5974808+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-right, context Some("Editor")

Caused by:
    no action type registered for editor::SelectToNextWordEnd
2024-08-14T13:35:03.5976124+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-z, context Some("Editor")

Caused by:
    no action type registered for editor::Redo
2024-08-14T13:35:03.5977456+02:00 [ERROR] invalid binding value for keystroke ctrl-up, context Some("Editor")

Caused by:
    no action type registered for editor::LineUp
2024-08-14T13:35:03.5978789+02:00 [ERROR] invalid binding value for keystroke ctrl-v, context Some("Editor")

Caused by:
    no action type registered for editor::Paste
2024-08-14T13:35:03.598021+02:00 [ERROR] invalid binding value for keystroke ctrl-x, context Some("Editor")

Caused by:
    no action type registered for editor::Cut
2024-08-14T13:35:03.5981615+02:00 [ERROR] invalid binding value for keystroke ctrl-y, context Some("Editor")

Caused by:
    no action type registered for editor::Redo
2024-08-14T13:35:03.5982968+02:00 [ERROR] invalid binding value for keystroke ctrl-z, context Some("Editor")

Caused by:
    no action type registered for editor::Undo
2024-08-14T13:35:03.5984321+02:00 [ERROR] invalid binding value for keystroke delete, context Some("Editor")

Caused by:
    no action type registered for editor::Delete
2024-08-14T13:35:03.5985672+02:00 [ERROR] invalid binding value for keystroke down, context Some("Editor")

Caused by:
    no action type registered for editor::MoveDown
2024-08-14T13:35:03.5987054+02:00 [ERROR] invalid binding value for keystroke end, context Some("Editor")

Caused by:
    no action type registered for editor::MoveToEndOfLine
2024-08-14T13:35:03.5988386+02:00 [ERROR] invalid binding value for keystroke escape, context Some("Editor")

Caused by:
    no action type registered for editor::Cancel
2024-08-14T13:35:03.5989719+02:00 [ERROR] invalid binding value for keystroke home, context Some("Editor")

Caused by:
    no action type registered for editor::MoveToBeginningOfLine
2024-08-14T13:35:03.5991109+02:00 [ERROR] invalid binding value for keystroke left, context Some("Editor")

Caused by:
    no action type registered for editor::MoveLeft
2024-08-14T13:35:03.5992444+02:00 [ERROR] invalid binding value for keystroke pagedown, context Some("Editor")

Caused by:
    no action type registered for editor::MovePageDown
2024-08-14T13:35:03.5993808+02:00 [ERROR] invalid binding value for keystroke pageup, context Some("Editor")

Caused by:
    no action type registered for editor::MovePageUp
2024-08-14T13:35:03.5995156+02:00 [ERROR] invalid binding value for keystroke right, context Some("Editor")

Caused by:
    no action type registered for editor::MoveRight
2024-08-14T13:35:03.5996504+02:00 [ERROR] invalid binding value for keystroke shift-backspace, context Some("Editor")

Caused by:
    no action type registered for editor::Backspace
2024-08-14T13:35:03.5997853+02:00 [ERROR] invalid binding value for keystroke shift-delete, context Some("Editor")

Caused by:
    no action type registered for editor::Cut
2024-08-14T13:35:03.5999244+02:00 [ERROR] invalid binding value for keystroke shift-down, context Some("Editor")

Caused by:
    no action type registered for editor::SelectDown
2024-08-14T13:35:03.6000619+02:00 [ERROR] invalid binding value for keystroke shift-end, context Some("Editor")

Caused by:
    no action type registered for editor::SelectToEndOfLine
2024-08-14T13:35:03.6001938+02:00 [ERROR] invalid binding value for keystroke shift-home, context Some("Editor")

Caused by:
    no action type registered for editor::SelectToBeginningOfLine
2024-08-14T13:35:03.6003303+02:00 [ERROR] invalid binding value for keystroke shift-insert, context Some("Editor")

Caused by:
    no action type registered for editor::Paste
2024-08-14T13:35:03.6004666+02:00 [ERROR] invalid binding value for keystroke shift-left, context Some("Editor")

Caused by:
    no action type registered for editor::SelectLeft
2024-08-14T13:35:03.6006021+02:00 [ERROR] invalid binding value for keystroke shift-pagedown, context Some("Editor")

Caused by:
    no action type registered for editor::SelectPageDown
2024-08-14T13:35:03.6007348+02:00 [ERROR] invalid binding value for keystroke shift-pageup, context Some("Editor")

Caused by:
    no action type registered for editor::SelectPageUp
2024-08-14T13:35:03.6008688+02:00 [ERROR] invalid binding value for keystroke shift-right, context Some("Editor")

Caused by:
    no action type registered for editor::SelectRight
2024-08-14T13:35:03.6010056+02:00 [ERROR] invalid binding value for keystroke shift-tab, context Some("Editor")

Caused by:
    no action type registered for editor::TabPrev
2024-08-14T13:35:03.6011374+02:00 [ERROR] invalid binding value for keystroke shift-up, context Some("Editor")

Caused by:
    no action type registered for editor::SelectUp
2024-08-14T13:35:03.6012706+02:00 [ERROR] invalid binding value for keystroke tab, context Some("Editor")

Caused by:
    no action type registered for editor::Tab
2024-08-14T13:35:03.6014037+02:00 [ERROR] invalid binding value for keystroke up, context Some("Editor")

Caused by:
    no action type registered for editor::MoveUp
2024-08-14T13:35:03.6015373+02:00 [ERROR] invalid binding value for keystroke alt-z, context Some("Editor && mode == full")

Caused by:
    no action type registered for editor::ToggleSoftWrap
2024-08-14T13:35:03.6016694+02:00 [ERROR] invalid binding value for keystroke ctrl-<, context Some("Editor && mode == full")

Caused by:
    no action type registered for assistant::InsertIntoEditor
2024-08-14T13:35:03.6018028+02:00 [ERROR] invalid binding value for keystroke ctrl->, context Some("Editor && mode == full")

Caused by:
    no action type registered for assistant::QuoteSelection
2024-08-14T13:35:03.6019341+02:00 [ERROR] invalid binding value for keystroke ctrl-alt-e, context Some("Editor && mode == full")

Caused by:
    no action type registered for editor::SelectEnclosingSymbol
2024-08-14T13:35:03.6020722+02:00 [ERROR] invalid binding value for keystroke ctrl-enter, context Some("Editor && mode == full")

Caused by:
    no action type registered for editor::NewlineAbove
2024-08-14T13:35:03.602238+02:00 [ERROR] invalid binding value for keystroke ctrl-f, context Some("Editor && mode == full")

Caused by:
    no action type registered for buffer_search::Deploy
2024-08-14T13:35:03.6023717+02:00 [ERROR] invalid binding value for keystroke ctrl-h, context Some("Editor && mode == full")

Caused by:
    no action type registered for buffer_search::Deploy
2024-08-14T13:35:03.6025086+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-enter, context Some("Editor && mode == full")

Caused by:
    no action type registered for editor::NewlineBelow
2024-08-14T13:35:03.6026445+02:00 [ERROR] invalid binding value for keystroke enter, context Some("Editor && mode == full")

Caused by:
    no action type registered for editor::Newline
2024-08-14T13:35:03.6027785+02:00 [ERROR] invalid binding value for keystroke shift-enter, context Some("Editor && mode == full")

Caused by:
    no action type registered for editor::Newline
2024-08-14T13:35:03.6029127+02:00 [ERROR] invalid binding value for keystroke alt-[, context Some("Editor && mode == full && inline_completion")

Caused by:
    no action type registered for editor::PreviousInlineCompletion
2024-08-14T13:35:03.6030594+02:00 [ERROR] invalid binding value for keystroke alt-], context Some("Editor && mode == full && inline_completion")

Caused by:
    no action type registered for editor::NextInlineCompletion
2024-08-14T13:35:03.6031937+02:00 [ERROR] invalid binding value for keystroke ctrl-right, context Some("Editor && mode == full && inline_completion")

Caused by:
    no action type registered for editor::AcceptPartialInlineCompletion
2024-08-14T13:35:03.6033289+02:00 [ERROR] invalid binding value for keystroke alt-\, context Some("Editor && !inline_completion")

Caused by:
    no action type registered for editor::ShowInlineCompletion
2024-08-14T13:35:03.6034622+02:00 [ERROR] invalid binding value for keystroke ctrl-enter, context Some("Editor && mode == auto_height")

Caused by:
    no action type registered for editor::Newline
2024-08-14T13:35:03.6035967+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-enter, context Some("Editor && mode == auto_height")

Caused by:
    no action type registered for editor::NewlineBelow
2024-08-14T13:35:03.6037292+02:00 [ERROR] invalid binding value for keystroke shift-enter, context Some("Editor && mode == auto_height")

Caused by:
    no action type registered for editor::Newline
2024-08-14T13:35:03.6038617+02:00 [ERROR] invalid binding value for keystroke ctrl-c, context Some("Markdown")

Caused by:
    no action type registered for markdown::Copy
2024-08-14T13:35:03.6039992+02:00 [ERROR] invalid binding value for keystroke alt-m, context Some("AssistantPanel")

Caused by:
    no action type registered for assistant::ToggleModelSelector
2024-08-14T13:35:03.6041336+02:00 [ERROR] invalid binding value for keystroke ctrl-g, context Some("AssistantPanel")

Caused by:
    no action type registered for search::SelectNextMatch
2024-08-14T13:35:03.6042666+02:00 [ERROR] invalid binding value for keystroke ctrl-k h, context Some("AssistantPanel")

Caused by:
    no action type registered for assistant::DeployHistory
2024-08-14T13:35:03.6044008+02:00 [ERROR] invalid binding value for keystroke ctrl-k l, context Some("AssistantPanel")

Caused by:
    no action type registered for assistant::DeployPromptLibrary
2024-08-14T13:35:03.604533+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-g, context Some("AssistantPanel")

Caused by:
    no action type registered for search::SelectPrevMatch
2024-08-14T13:35:03.6046671+02:00 [ERROR] invalid binding value for keystroke ctrl-n, context Some("PromptLibrary")

Caused by:
    no action type registered for prompt_library::NewPrompt
2024-08-14T13:35:03.6047977+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-s, context Some("PromptLibrary")

Caused by:
    no action type registered for prompt_library::ToggleDefaultPrompt
2024-08-14T13:35:03.6049339+02:00 [ERROR] invalid binding value for keystroke alt-enter, context Some("BufferSearchBar")

Caused by:
    no action type registered for search::SelectAllMatches
2024-08-14T13:35:03.6050686+02:00 [ERROR] invalid binding value for keystroke ctrl-f, context Some("BufferSearchBar")

Caused by:
    no action type registered for search::FocusSearch
2024-08-14T13:35:03.6052011+02:00 [ERROR] invalid binding value for keystroke ctrl-h, context Some("BufferSearchBar")

Caused by:
    no action type registered for search::ToggleReplace
2024-08-14T13:35:03.6053341+02:00 [ERROR] invalid binding value for keystroke ctrl-l, context Some("BufferSearchBar")

Caused by:
    no action type registered for search::ToggleSelection
2024-08-14T13:35:03.605465+02:00 [ERROR] invalid binding value for keystroke enter, context Some("BufferSearchBar")

Caused by:
    no action type registered for search::SelectNextMatch
2024-08-14T13:35:03.6055982+02:00 [ERROR] invalid binding value for keystroke escape, context Some("BufferSearchBar")

Caused by:
    no action type registered for buffer_search::Dismiss
2024-08-14T13:35:03.6057298+02:00 [ERROR] invalid binding value for keystroke shift-enter, context Some("BufferSearchBar")

Caused by:
    no action type registered for search::SelectPrevMatch
2024-08-14T13:35:03.6058616+02:00 [ERROR] invalid binding value for keystroke tab, context Some("BufferSearchBar")

Caused by:
    no action type registered for buffer_search::FocusEditor
2024-08-14T13:35:03.6059966+02:00 [ERROR] invalid binding value for keystroke ctrl-enter, context Some("BufferSearchBar && in_replace")

Caused by:
    no action type registered for search::ReplaceAll
2024-08-14T13:35:03.6061283+02:00 [ERROR] invalid binding value for keystroke enter, context Some("BufferSearchBar && in_replace")

Caused by:
    no action type registered for search::ReplaceNext
2024-08-14T13:35:03.6062638+02:00 [ERROR] invalid binding value for keystroke down, context Some("BufferSearchBar && !in_replace > Editor")

Caused by:
    no action type registered for search::NextHistoryQuery
2024-08-14T13:35:03.6063981+02:00 [ERROR] invalid binding value for keystroke up, context Some("BufferSearchBar && !in_replace > Editor")

Caused by:
    no action type registered for search::PreviousHistoryQuery
2024-08-14T13:35:03.6065293+02:00 [ERROR] invalid binding value for keystroke alt-ctrl-g, context Some("ProjectSearchBar")

Caused by:
    no action type registered for search::ToggleRegex
2024-08-14T13:35:03.6066626+02:00 [ERROR] invalid binding value for keystroke alt-ctrl-x, context Some("ProjectSearchBar")

Caused by:
    no action type registered for search::ToggleRegex
2024-08-14T13:35:03.6067925+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-f, context Some("ProjectSearchBar")

Caused by:
    no action type registered for search::FocusSearch
2024-08-14T13:35:03.6069221+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-h, context Some("ProjectSearchBar")

Caused by:
    no action type registered for search::ToggleReplace
2024-08-14T13:35:03.6070562+02:00 [ERROR] invalid binding value for keystroke escape, context Some("ProjectSearchBar")

Caused by:
    no action type registered for project_search::ToggleFocus
2024-08-14T13:35:03.6071863+02:00 [ERROR] invalid binding value for keystroke down, context Some("ProjectSearchBar > Editor")

Caused by:
    no action type registered for search::NextHistoryQuery
2024-08-14T13:35:03.6073184+02:00 [ERROR] invalid binding value for keystroke up, context Some("ProjectSearchBar > Editor")

Caused by:
    no action type registered for search::PreviousHistoryQuery
2024-08-14T13:35:03.6074496+02:00 [ERROR] invalid binding value for keystroke ctrl-alt-enter, context Some("ProjectSearchBar && in_replace > Editor")

Caused by:
    no action type registered for search::ReplaceAll
2024-08-14T13:35:03.607581+02:00 [ERROR] invalid binding value for keystroke enter, context Some("ProjectSearchBar && in_replace > Editor")

Caused by:
    no action type registered for search::ReplaceNext
2024-08-14T13:35:03.6077165+02:00 [ERROR] invalid binding value for keystroke alt-ctrl-g, context Some("ProjectSearchView")

Caused by:
    no action type registered for search::ToggleRegex
2024-08-14T13:35:03.6078469+02:00 [ERROR] invalid binding value for keystroke alt-ctrl-x, context Some("ProjectSearchView")

Caused by:
    no action type registered for search::ToggleRegex
2024-08-14T13:35:03.6079789+02:00 [ERROR] invalid binding value for keystroke ctrl-shift-h, context Some("ProjectSearchView")

Caused by:
    no action type registered for search::ToggleReplace
2024-08-14T13:35:03.6081104+02:00 [ERROR] invalid binding value for keystroke escape, context Some("ProjectSearchView")

Caused by:
    no action type registered for project_search::ToggleFocus
2024-08-14T13:35:03.6082408+02:00 [ERROR] invalid binding value for keystroke alt-c, context Some("Pane")

Caused by:
    no action type registered for search::ToggleCaseSensitive
2024-08-14T13:35:03.6083716+02:00 [ERROR] invalid binding value for keystroke alt-ctrl-f, context Some("Pane")

Caused by:
    no action type registered for project_search::ToggleFilters
2024-08-14T13:35:03.608502+02:00 [ERROR] invalid binding value for keystroke alt-ctrl-shift-w, context Some("Pane")

Caused by:
    no action type registered for workspace::CloseInactiveTabsAndPanes
2024-08-14T13:35:03.6086328+02:00 [ERROR] invalid binding value for keystroke alt-ctrl-t, context Some("Pane")

--- a/rust-toolchain.toml
+++ b/rust-toolchain.toml
@@ -2,4 +2,4 @@
 channel = "1.80"
 profile = "minimal"
 components = [ "rustfmt", "clippy" ]
-targets = [ "x86_64-apple-darwin", "aarch64-apple-darwin", "x86_64-unknown-linux-gnu", "wasm32-wasi" ]
+targets = [ "x86_64-apple-darwin", "aarch64-apple-darwin", "x86_64-unknown-linux-gnu", "wasm32-wasi", "x86_64-pc-windows-msvc" ]

